language-support/java/codegen/src/it/java/com/digitalasset/ListTest.java

```diff
@@ -146,4 +149,78 @@ void listOfColorsFromProtobufValue() {
 
     }
 
+    @Test
+    void parameterizedListTestRoundTrip() {
```

Comment:
Can you move this test to a different PR independent from the introduction of Map?

Reply:
Good idea.